### DISCUSSION TITLE
Problem in running example 4_restart

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/FEPX#46] [Q&A] [answered]
Hello!
After successfully running the command run_example, I am getting an error in the visualization command while post-processing the example 4_restart. Please help me out with this.

**Comment by mkasemer** [comment, marked as answer]
Hello @sushdash,

It is difficult to diagnose the problem without specifics: the best place to start is to post the complete output of the simulation and the post-processing, to see what has gone wrong.

It is also difficult to say at this point, but the error could be with either FEPX or Neper. If the latter, it is best to post to the Neper discussion forum.

**Reply by sushdash** [reply]
Thank you for your response!
I reinstalled Neper using the latest source code available in GitHub and it worked!
